### PR TITLE
Secure login

### LINKED ISSUE #60: Accounting vulnerable against brute force
Number of login requests with wrong password is not limited => brute force is not prevented! Since the password length is min six characters online guessing can be successful really quickly. Proposal: max 5 wrong password requests, min 8 characters password.

### PR DESCRIPTION
**Pull request by audax**
Login throttle with 5 attempts per 1 minute

Password policy:
Minimum characters: 8
No common passwords
No user name as password
No numeric only passwords

Resolves #60

### CODE REVIEW

File: qabel_id/settings/base_settings.py [hunk 1]
```diff
@@ -68,6 +71,29 @@
     )
 }
 
+# Login security
+
+AXES_COOLOFF_TIME = datetime.timedelta(minutes=1)
```
Comment (thechauffeur) [inline qabel_id/settings/base_settings.py]: The description of this PR says per 10 seconds. This LOC says per 1 minute. I like 1 minute but please update the description.

**Comment by thechauffeur** [comment]
@audax is it possible to provide (unit) tests for the imbued limits in a simple way? If not I would suggest to put these tests into our infrastructure / load testing scripts. Please arrange with @roeslpa and @jan-schreib.

**Comment by audax** [comment]
I still need to integrate this with the django-rest-framework

**Comment by julianseeger** [comment]
works for me

**Pull request merged**